_posts/2018-04-28-hackillinois-2018-recap.markdown

```diff
@@ -1,28 +1,47 @@
 ---
 layout: post
 title: "Dreamt it. Built it. HackIllinois 2018, A Recap."
-date: 2018-04-28 00:00:00
+date: 2018-05-19 00:00:00
 categories: recap
 tags: HackIllinois2018
-image: /assets/article_images/hackillinois_2018.jpg
+image: /assets/article_images/2018_recap_cover.jpg
 author: Justin Yang
 ---
 
-This February, HackIllinois welcomed over a thousand students to the University of Illinois at Urbana-Champaign for an excitement-packed weekend to develop open-source projects.
+This February, HackIllinois welcomed over 1000 students to the University of Illinois at Urbana-Champaign for a developement-filled weekend where students from across the Midwest and the country gathered to make contributions to open source projects. Over the course of just 36 hours, attendees made signifigant contributions, making over [94 submissions](https://hackillinois-2018.devpost.com/submissions) by the end!
 
-HackIllinois 2018 kicked off Friday afternoon, with students lining up for registration and to chat with sponsors at the company expo. After everyone filled up on dinner from Panda Express and Kohinoor, they heard opening remarks from the Opening Ceremony. Attendees then broke out into two panels based on their interests: Web & Data and Systems where open-source mentors spoke about their projects and the open-source community in general.
+![opening ceremony](/assets/article_images/ceremony.jpg "HackIllinois 2018 kicks off!")
 
-Attendees, however, weren't just at HackIllinois to hear about open source. As the panels concluded, attendees formed into groups via the project matching sessions to work on open-source projects -- be it with open-source projects to improve popular open-source projects such as Rust and Freenode.
+With the focus on open source at HackIllinois, attendees spent the weekend learning about and building out new open source features and tools that tackled issues in many different fields. HackIllinois kicked off with panels on Web/Data and Systems/Miscellaneous after the Opening Ceremony. Mentors presented some of the projects students could join, after which students joined groups they were interested in.
```

Comment:
"Mentors presented their projects, after which students joined groups they were interested in."

Reply:
👍